server/extensions/input.go

```diff
@@ -0,0 +1,16 @@
+package extensions
+
+import (
+	"github.com/layer5io/meshery/server/machines"
+	"github.com/layer5io/meshkit/broker"
+	"github.com/layer5io/meshkit/database"
+	"github.com/layer5io/meshkit/logger"
+)
+
+type ExtensionInput struct {
```

Comment:
Extension, huh?

Reply:
The PR extracts the constructs that are being used for the go plugin from `models` into its own package.